plutus-ledger-api/src/PlutusLedgerApi/Common/Versions.hs

```diff
@@ -70,6 +71,10 @@ builtinsIntroducedIn = Map.fromList [
           ]),
   ((PlutusV2, ProtocolVersion 7 0), Set.fromList [
           SerialiseData, VerifyEcdsaSecp256k1Signature, VerifySchnorrSecp256k1Signature
+          ]),
+  -- FIXME: Protocol version TBD
+  ((PlutusV3, ProtocolVersion 8 0), Set.fromList [
```

Comment:
I'd probably just not include this until there's something to put in it.

Reply:
Also, we don't know that we're enabling V3 in Chang. We may well not be.

Reply:
This sort of thing makes me again want to have a protocol-version check at the start of the evaluation functions too...